### ISSUE TITLE
Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by sagarthakore** [rpelaez/IssueCardPrinterPRO#1]
I am getting the following error message when I try to print a card -

![issue_jira_print](https://user-images.githubusercontent.com/16065570/42701456-9eb31d18-8694-11e8-9f62-d6386e530df7.PNG)

**Comment by rpelaez** [comment]
Just remove from your Google Chrome browser the extension "Issue Card Printer for Jira Cloud" and install the "Issue Card Printer PRO for Jira Cloud". https://chrome.google.com/webstore/detail/issue-card-printer-pro-fo/fccoaoljiaiaogkcegghcgjdgfaknpla?hl=en

Are you using Jira Cloud or Jira Server? 
Regards

**Comment by sagarthakore** [comment]
I'm sure I am not using the Chrome extension. It is a Jira plugin. Also, I'm using Jira Server and not Jira Cloud.

Thanks

**Comment by rpelaez** [comment]
Hi Sagar,

Thanks for clarify, then you can raise a support ticket in this web: https://jirasupport.atlassian.net/servicedesk/customer/portal/1

It's strange because this error in the new versions was fixed some months ago.
In the ticket please provide version of Jira and the version of the plugin

Best regards

**Comment by sagarthakore** [comment]
I'm using version 3.1 of the plugin. If this issue was fixed in a later version, I can request my Jira admin to update to the latest version. If not, then I'll open a support ticket. 

Thanks

**Comment by rpelaez** [comment]
Yes 3.1 is the latest version, then please raise a new support ticket with the Jira version and the browser version, OS, and the sequence/steps to reproduce the error please. If you can, please add a screenshot of the error with the Development tools (of your browser) activated to see the error raised and the line of code. Many thanks and sorry for the incidence
